compiler/ast.nim

```diff
@@ -303,6 +303,7 @@ type
     sfSingleUsedTemp  # For temporaries that we know will only be used once
     sfNoalias         # 'noalias' annotation, means C's 'restrict'
     sfEffectsDelayed  # an 'effectsDelayed' parameter
+    sfEnumIsUnsigned  # enum field is unsigned
```

Comment:
Why do you need both a sym flag and a type flag for this?

Reply:
Ah, I misinterpreted the output for `debug` on PSym.  I see now I could get the type flags from `v.typ` as long as the value isn't nil, so the sym flag isn't necessary! I'll repush after more testing. Thanks for reviewing this!

Reply:
You're most welcome.